lib/ObisMeasurement.d.ts

```diff
@@ -0,0 +1,19 @@
+export declare class ObisMeasurement {
```

Comment:
This class is missing all instance methods. Or are they not being used outside the class itself?

Reply:
I spared out these functions on purpose, since they are mentioned at no point in documentation and their functionality is completely summarized by the `resolveObisName` function.

Unfortunately, there's no way to express public- and privateness in the old-fashioned JavaScript syntax, so maybe @Apollon77 can solve this mystery? 😄

Reply:
In fact the callback on new values will get an array of ObisMeasurement instances. So each instance of this represents one returned value and allows to access it. In fact all get Methods (+idToString) are the relevants ones for users of the Library to get summary access or access to single values or parts, so best would be to expose all methods.

the example in README uses idToString and valueToString. The ioBroker update als getRawValue and more ...